doc_src/en/HowTo_SetUpTeamProject.xml

```diff
@@ -243,6 +244,46 @@
 	</orderedlist>
   </section>
 
+  <section id="how.to.manage.team.project">
+  <title id="how.to.manage.team.project.title">Manage a team project</title>
+
+  <para>Team projects involve a number of management tasks not directly related
+  to translation. Project managers may find it advantageous—or even necessary—to
+  handle such tasks directly from the VCS repository, particularly in cases where
+  the project manager only acts as a coordinator and is not involved in the translation
+  process itself.</para>
+    <variablelist>
+	  <varlistentry>
+	    <term>Deleting files</term>
+	    <listitem>
+		  <para>OmegaT does not provide a mechanism to delete files from a team project.
+		  If any shared team project files have become unnecessary, the project manager
+		  <emphasis>must</emphasis> delete them from the VCS repository.</para>
+	    </listitem>
+	  </varlistentry>
+	  <varlistentry>
+		<term>Adding files</term>
+		<listitem>
+		  <para>Although OmegaT provides a mechanism for adding translation source and
+		  target files to the repository, the project manager must add any other files,
+		  such as dictionaries, glossaries or reference translation memories, from the
+		  VCS repository.</para>
+		</listitem>
+	  </varlistentry>
+	  <varlistentry>
+		<term>Checking project statistics</term>
+		<listitem>
+		  <para>This only applies to workflows that ask translators to commit the translated
```

Comment:
Why ?

Reply:
Because that's essentially the way the feature request and the implementation work. Statistics are only committed when target files are committed.

I'll have to expand on that more, though. Maybe add a short section on the difference between managing projects as:
- one of the translators involved, with a copy of the project on their system
- purely a coordinator not involved in the actual translating, and working directly with the repository and without a copy of the OmegaT project on their system.